README.md

```diff
@@ -172,6 +172,46 @@ TID is a 32-bit integer to identify threads created with `wasi_thread_spawn`.
     For example, it can be used to indicate the main thread, which doesn't
     have a TID in the current version of this proposal.
 
+### Process
+
+* A process is a group of threads.
+
+* The main thread starts with a process which only contains
+  the main thread.
+
+* Threads created by a thread in a process using `wasi_thread_spawn`
+  is added to the process.
+
+* When a thread is terminated, it's removed from the process.
+
+### Voluntary thread termination
+
+A thread can terminate itself voluntarily by returning from
+`wasi_thread_start`.
+
+### Changes to WASI `proc_exit`
+
+With this proposal, the `proc_exit` function takes extra responsibility
+to terminate all threads in the process, not only the calling one.
+
+Any of threads in the process can call `proc_exit`.
+
+### Traps
+
+When a thread caused a trap, it terminates all threads in the process
+similarly to `proc_exit`.
+
+### Process exit status
+
+If one or more threads call WASI `proc_exit` or raise a trap,
```

Comment:
Does it include calling it `proc_exit` from the main thread?

Reply:
Yes